packages/cells/src/widget.ts

```diff
@@ -480,7 +480,7 @@ class CodeCell extends Cell {
     let model = this.model;
 
     // Code cells should not wrap lines.
-    this.editor.wordWrap = false;
+    this.editor.setOption('wordWrap', false);
```

Comment:
After trying it out interactively, I think `lineWrap` or `wrapLines` is a better way of describing what this does. `wordWrap` has a connotation of wrapping the actual words internally.